diff --git a/node_modules/@cornerstonejs/dicom-image-loader/dist/esm/shared/decoders/decodeJPEG2000.js b/node_modules/@cornerstonejs/dicom-image-loader/dist/esm/shared/decoders/decodeJPEG2000.js
index ef7334f..b6f2374 100644
--- a/node_modules/@cornerstonejs/dicom-image-loader/dist/esm/shared/decoders/decodeJPEG2000.js
+++ b/node_modules/@cornerstonejs/dicom-image-loader/dist/esm/shared/decoders/decodeJPEG2000.js
@@ -1,5 +1,5 @@
 import openJpegFactory from '@cornerstonejs/codec-openjpeg/decodewasmjs';
-const openjpegWasm = new URL('@cornerstonejs/codec-openjpeg/decodewasm', import.meta.url);
+const openjpegWasm = new URL('/assets/codecs/openjpegwasm_decode.wasm', window.location.origin);
 const local = {
     codec: undefined,
     decoder: undefined,
